### PR TITLE
Implement 'amp-ad' type: 'ligatus'

### PR DESCRIPTION
**Pull request by dniggemann**
Hello,

I would like to request the support for a new 3P ad service.
The intention is to allow 'amp-ad built-in tag' to request ads from ligatus ad server.

Thanks!

### CODE REVIEW

**Comment by zhouyx** [comment]
@dniggemann Thanks for the Pull Request
We have a [renderStart API](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis) that helps AMP to provide better ad loading experience. 
Could you take a look and implement it (renderStart + noContentAvailable)?
Let me know if there's any question.

**Comment by dniggemann** [comment]
Hello @zhouyx , thanks for your advice. We definitely intend to implement the renderStart- and noContentAvailable API.
Unless these APIs are not strictly necessary, and since it’s also planned to support 'ad resizing' in the immediate future, would it be okay to consider the implementation in the next pull request?
I would be glad, if a (very) basic integration gets approved first.

File: ads/ligatus.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+-->
+
+# Ligatus
+
+## Example
+
+```html
+<amp-ad width=300 height=250
+    type="ligatus"
+    src="https://a-ssl.ligatus.com/?ids=88443&t=js&s=1&bc=2">
```
Comment (zhouyx) [inline ads/ligatus.md]: not related. why is the `&` displayed in red? any ideas? 😄

**Comment by zhouyx** [comment]
@dniggemann The PR looks great to me!
But please plan to add support to render-start in the future to improve your ad loading experience.
Ping @lannka  to take a final look

**Pull request merged**